src/Model/FocusManager.re

```diff
@@ -0,0 +1,26 @@
+open Oni_Core;
+
+module Option = Utility.Option;
+
+let push = (focusable: Focus.focusable, state: State.t) =>
+  switch (focusable) {
+  | Quickmenu
```

Comment:
At first I wasn't sure why `Quickmenu` and `Wildmenu` were special cased here - but then I re-read the PR description and the `current` method and see - those receive implicit focus based on the `state`, and thus shouldn't explicitly need to be pushed / popped.

Reply:
Exactly, although the more important invariant to maintain here is that it shouldn't be possible for _anything else_ to steal focus when the menus are visible, which would be much harder to ensure without this.